scaleway/data_source_rdb_acl_test.go

```diff
@@ -10,32 +10,27 @@ import (
 func TestAccScalewayDataSourceRDBAcl_Basic(t *testing.T) {
 	tt := NewTestTools(t)
 	defer tt.Cleanup()
-	instanceName := "TestAccScalewayDataSourceRDBAcl_Basic"
+	instanceName := "data-source-rdb-acl-basic"
 	resource.ParallelTest(t, resource.TestCase{
 		PreCheck:          func() { testAccPreCheck(t) },
 		ProviderFactories: tt.ProviderFactories,
 		CheckDestroy:      testAccCheckScalewayRdbInstanceDestroy(tt),
 		Steps: []resource.TestStep{
 			{
 				Config: fmt.Sprintf(`
-					resource "scaleway_rdb_instance" "main" {
+					resource "scaleway_rdb_instance" "rdbAcl" {
```

Comment:
why this rename is needed? We already know that it is rdbacl because of the name of the test. Besides, why is acl deleted?